.gitignore

```diff
@@ -1,3 +1,8 @@
 Gemfile.lock
 _site
 .DS_Store
+
+# Ignore generated output on the gh-pages branch to make branch-switching less
+# likely to accidentally include inappropriate files.
+target
+.idea
```

Comment:
I'd prefer `/.idea` here for specificity; I doubt anyone would put their IntelliJ project in a subdirectory.